package.json

```diff
@@ -119,6 +120,7 @@
     "react-test-renderer": "^16.6.3",
     "resolve": "1.8.1",
     "sass-loader": "7.1.0",
+    "serve": "^10.1.2",
```

Comment:
Does this have a specified port or maybe more specifically, is the port number configurable?

Reply:
It serves from port 5000 by default. You can configure it to serve from any port with the -l flag. running `serve -h` gives all of the possible configuration options.